### DISCUSSION TITLE
How to use custom .msp library in the Lipidomics workflow in MSDIAL 4.9.221218?

### DISCUSSION THREAD

**Discussion by VlCapka** [systemsomicslab/MsdialWorkbench#275] [Q&A]
Hello all,
We made a custom .msp lipidomics library and would like to use it in MSDIAL 4.9.221218. However, in the lipidomics workflow, the program does not allow us to select the .msp file. The select button takes you to lipid subclass slection but does not allow us to browse to the library file. We tried to paste the path to the .msp file in the selection box but upon processing, this link is ignored and a default, build-in library is used instead. 
Does anyone have any advice on how to solve this problem?
Thank you,

**Comment by ondrej-kuda** [comment]
Yo don't. Lipidomics library comes from the internal LipidBlast. If you have your own lipidomics MSP, just start a Metabolomics project and use your lipidomics MSP.
